package.json

```diff
@@ -1,80 +1,93 @@
-            }
-        ]
-    },
-    "dependencies": {
-        "shell-quote": "^1.7.2",
-        "tmp-promise": "^3.0.2",
-        "vscode-languageclient": "^7.0.0",
-        "vscode-languageserver": "^7.0.0",
-        "vscode-languageserver-textdocument": "^1.0.1",
-        "vscode-uri": "^3.0.2",
-        "web-tree-sitter": "^0.19.3"
-    },
-    "devDependencies": {
-        "@types/mocha": "^8.2.2",
-        "@types/node": "^15.0.2",
-        "@types/shell-quote": "^1.7.0",
-        "@types/tmp": "^0.2.0",
-        "@types/vscode": "^1.55.2",
-        "mocha": "^8.4.0",
-        "tree-sitter-cli": "^0.19.3",
-        "tree-sitter-teal": "https://github.com/euclidianAce/tree-sitter-teal/tarball/v0.0.4",
-        "typescript": "^4.2.4",
-        "vsce": "^1.88.0"
-    },
-    "scripts": {
-        "package": "vsce package",
-        "vscode:prepublish": "npm run build",
-        "build": "tsc -b && npm run generate-grammar && tree-sitter build-wasm node_modules/tree-sitter-teal && node scripts/post-build.js",
-        "watch": "tsc -b -w",
-        "test": "mocha out/test/**/*.spec.js",
-        "generate-grammar": "cd node_modules/tree-sitter-teal && tree-sitter generate"
-    }
-}
+	"name": "vscode-teal",
+	"displayName": "Teal",
+	"description": "Teal language support for Visual Studio Code",
+	"version": "0.7.6",
+	"license": "MIT",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/teal-language/vscode-teal.git"
+	},
+	"engines": {
+		"vscode": "^1.52.0"
+	},
+	"categories": [
+		"Programming Languages"
+	],
+	"icon": "assets/teal_logo.png",
+	"main": "./out/client/extension",
+	"activationEvents": [
+		"onLanguage:teal"
+	],
+	"publisher": "pdesaulniers",
+	"contributes": {
+		"languages": [
+			{
+				"id": "teal",
+				"aliases": [
+					"Teal",
+					"tl"
+				],
+				"extensions": [
+					".tl"
+				],
+				"configuration": "./language-configuration.json"
+			}
+		],
+		"grammars": [
+			{
+				"language": "teal",
+				"scopeName": "source.teal",
+				"path": "./syntaxes/teal.tmLanguage.json"
+			}
+		],
+		"snippets": [
+			{
+				"language": "teal",
+				"path": "./snippets/teal.json"
+			}
+		],
+		"iconThemes": [
+		  {
+			"id": "teal-icons",
+			"label": "Teal Icons",
+			"path": "./fileicons/teal-icon-theme.json"
+		  }
+		]
+	},
+	"dependencies": {
+		"shell-quote": "^1.7.2",
+		"tmp-promise": "^3.0.2",
+		"vscode-languageclient": "^7.0.0",
+		"vscode-languageserver": "^7.0.0",
+		"vscode-languageserver-textdocument": "^1.0.1",
+		"vscode-uri": "^3.0.2",
+		"web-tree-sitter": "^0.18.0"
+	},
+	"devDependencies": {
+		"@types/mocha": "^8.2.0",
+		"@types/node": "^14.14.22",
+		"@types/shell-quote": "^1.7.0",
+		"@types/tmp": "^0.2.0",
+		"@types/vscode": "^1.52.0",
+		"mocha": "^8.3.0",
+		"ovsx": "0.1.0-next.a9154dc",
+		"tree-sitter-cli": "^0.18.0",
+		"tree-sitter-teal": "github:euclidianAce/tree-sitter-teal#f6eb1b2",
+		"typescript": "^4.1.3",
+		"vsce": "^1.85.0"
+	},
```

Comment:
It looks like the dependencies have been downgraded.

I suppose your changes are based on the latest .vsix version of the extension? Can you please base your changes on the current 'master' branch of the repo instead?

Reply:
Sorry about that.  I used the latest version from the extension.  I have updated it to be based off the current main branch.  Should be resolved now.